### ISSUE TITLE
Update insecure dependencies

### ISSUE THREAD

**Issue by zemlanin** [wix/yoshi#1584]
Running `npm audit` in repo root and in `packages/yoshi` returns *a lot* of insecure packages:

```
~/w/yoshi> npm config set registry https://registry.npmjs.org/
~/w/yoshi> npm i --package-lock-only
[...]

~/w/yoshi> npm audit
                       === npm audit security report ===
[...]
found 4 vulnerabilities (2 low, 2 high) in 924445 scanned packages
  run `npm audit fix` to fix 3 of them.
  1 vulnerability requires semver-major dependency updates.
```

```
~/w/yoshi/packages/yoshi> npm i --package-lock-only
[...]

~/w/yoshi/packages/yoshi> npm audit
                       === npm audit security report ===
[...]
found 106 vulnerabilities (16 low, 45 moderate, 40 high, 5 critical) in 1936214 scanned packages
  run `npm audit fix` to fix 9 of them.
  38 vulnerabilities require semver-major dependency updates.
  59 vulnerabilities require manual review. See the full report for details.
```

I _think_ it should be fixed

_(also "Wow, millions of dependencies")_ 🤯

**Comment by zemlanin** [comment]
_further activity occurring_

**Comment by zemlanin** [comment]
![](http://code.oursky.com/wp-content/uploads/2015/11/repeatedly-click.gif)

**Comment by ranyitz** [comment]
Solved on #1756
